Makefile

```diff
@@ -23,6 +23,7 @@ DESTDIR ?= /usr/local
 VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
 REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
 PACKAGE=github.com/containerd/containerd
+CGO ?= 0
```

Comment:
Curious; why not `GCO_ENABLED`, and remove it from the lines below?

Reply:
This is only for the shims, maybe comment it as such? The regular `CGO_ENABLED` value will be used for the other binaries